sql-odbc/src/sqlodbc/opensearch_info.cpp

```diff
@@ -86,7 +86,7 @@ const std::unordered_map< std::string, int > data_name_data_type_map = {
     {OPENSEARCH_TYPE_NAME_SCALED_FLOAT, SQL_DOUBLE},
     {OPENSEARCH_TYPE_NAME_KEYWORD, SQL_WVARCHAR},
     {OPENSEARCH_TYPE_NAME_TEXT, SQL_WVARCHAR},
-    {OPENSEARCH_TYPE_NAME_DATE, SQL_TYPE_TIMESTAMP},
+    {OPENSEARCH_TYPE_NAME_TIMESTAMP, SQL_TYPE_TIMESTAMP},
```

Comment:
Shouldn't you keep the date mapping to timestamp here as well?

Reply:
Yes, I've reverted it back.